src/makeRobot.js

```diff
@@ -37,8 +37,61 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
```

Comment:
```suggestion
    name, 
    coords: {
      x: 0,
      y: 0,
    },
```
define `name`, `wheels` and `version` inside, and it would be better to move them on top before `get info() `